lang/ilias_en.lang

```diff
@@ -5260,6 +5261,7 @@ common#:#pd_items_news#:#Include News of Personal Items
 common#:#pdf_export#:#PDF Export
 common#:#perm_settings#:#Permissions
 common#:#perma_link#:#Permanent Link
+common#:#perma_link_copied#:#Link was copied to clipboard.
```

Comment:
THis got lost in my first review: Link to this page has been copied to the clipboard.